handler/release_handler.go

```diff
@@ -104,7 +100,13 @@ func updateReleaseNotes(client *github.Client, owner, repo, latestTag string) er
 		output = output + fmt.Sprintf("%s %s%s\n", id, title, author)
 	}
 
-	output = fmt.Sprintf("%s\nGenerated by [Derek](https://github.com/alexellis/derek/)\n", output)
+	var releaseDiff string
+	if workingReleases.PreviousTag != "" {
+		releaseDiff = fmt.Sprintf("Changes between `%[1]s` and `%[2]s`: https://github.com/%[3]s/%[4]s/compare/%[1]s...%[2]s", workingReleases.PreviousTag, workingReleases.CurrentTag, owner, repo)
```

Comment:
I think I'd like to see it as: `Changes: 0.1.0..0.2.0` for instance. GitHub should make it into a link automatically, but if it doesn't you still have the longer link to fall back on.

To try variations, you can edit the text in a release and update it as much as you want outside of using Derek, then when you have it like you want, update your deployment.